src/main/java/it/pn/frontend/e2e/common/RecapitiDestinatarioPage.java

```diff
@@ -369,6 +381,17 @@ public boolean siControllaNonPresenzaPEC() {
         }
     }
 
+    public boolean siControllaPresenzaPEC() {
+        try {
+            By pecField = By.id("legalContacts");
+            this.getWebDriverWait(10).until(ExpectedConditions.visibilityOfElementLocated(pecField));
```

Comment:
The element is already declared at the top of the file:
https://github.com/pagopa/pn-frontend-e2e/blob/133de6c62d0b35c4ef53e27c90ae3bd486431217/src/main/java/it/pn/frontend/e2e/common/RecapitiDestinatarioPage.java#L71-L72